common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,126 @@
+"""OLI analytics service event tracker backend."""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from django.contrib.auth.models import User
+from student.models import anonymous_id_for_user
```

Comment:
I _think_ we need an extra newline before this line? The guide being that imports should be included in 3 separate stanzas:

```python
import standard_library

import third_party_libraries

import local_modules
```